mempool/src/counters.rs

```diff
@@ -310,7 +310,7 @@ pub static CORE_MEMPOOL_TXN_CONSENSUS_PULLED: Lazy<Histogram> = Lazy::new(|| {
     register_histogram!(
         "aptos_core_mempool_txn_consensus_pulled",
         "Number of times a txn was pulled from core mempool by consensus",
-        TXN_CONSENSUS_PULLED_BUCKETS.to_vec()
+        &["bucket"]
```

Comment:
You'll need to replace `register_histogram` with `register_histogram_vec`

And let's keep the histogram buckets override as the numbers should be better than the default (not to be confused with the bucket label)

```
register_histogram_vec!(..., &["bucket"], TXN_CONSENSUS_PULLED_BUCKETS.to_vec())

Reply:
Done.